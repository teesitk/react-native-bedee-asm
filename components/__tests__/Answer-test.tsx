import * as React from 'react';
import renderer from 'react-test-renderer';
import Answer from '../Answers';


it(`renders correctly`, () => {
  const tree = renderer.create(<Answer data={[]} />).toJSON();

  expect(tree).toMatchSnapshot();
});
