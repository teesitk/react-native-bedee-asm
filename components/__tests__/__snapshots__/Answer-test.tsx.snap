// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View>
  <View
    style={
      {
        "alignItems": "center",
        "backgroundColor": "#ffffff",
        "borderBottomWidth": 0.5,
        "borderColor": "#bcbbc1",
        "flexDirection": "row",
        "padding": 14,
      }
    }
    testID="padView"
  >
    <View
      style={
        [
          {
            "alignItems": "flex-start",
            "flex": 1,
            "justifyContent": "center",
          },
          undefined,
        ]
      }
      theme={
        {
          "colors": {
            "black": "#242424",
            "disabled": "hsl(208, 8%, 90%)",
            "divider": "#bcbbc1",
            "error": "#ff190c",
            "grey0": "#393e42",
            "grey1": "#43484d",
            "grey2": "#5e6977",
            "grey3": "#86939e",
            "grey4": "#bdc6cf",
            "grey5": "#e1e8ee",
            "greyOutline": "#bbb",
            "platform": {
              "android": {
                "error": "#f44336",
                "grey": "rgba(0, 0, 0, 0.54)",
                "primary": "#2196f3",
                "searchBg": "#dcdce1",
                "secondary": "#9C27B0",
                "success": "#4caf50",
                "warning": "#ffeb3b",
              },
              "default": {
                "error": "#ff3b30",
                "grey": "#7d7d7d",
                "primary": "#007aff",
                "searchBg": "#dcdce1",
                "secondary": "#5856d6",
                "success": "#4cd964",
                "warning": "#ffcc00",
              },
              "ios": {
                "error": "#ff3b30",
                "grey": "#7d7d7d",
                "primary": "#007aff",
                "searchBg": "#dcdce1",
                "secondary": "#5856d6",
                "success": "#4cd964",
                "warning": "#ffcc00",
              },
              "web": {
                "error": "#ff190c",
                "grey": "#393e42",
                "primary": "#2089dc",
                "searchBg": "#303337",
                "secondary": "#ca71eb",
                "success": "#52c41a",
                "warning": "#faad14",
              },
            },
            "primary": "#2089dc",
            "searchBg": "#303337",
            "secondary": "#ca71eb",
            "success": "#52c41a",
            "warning": "#faad14",
            "white": "#ffffff",
          },
        }
      }
    >
      <Text
        style={
          {
            "backgroundColor": "transparent",
            "color": "#242424",
            "fontSize": 17,
          }
        }
        testID="listItemTitle"
      >
        <Text
          style={
            [
              {
                "color": "#11181C",
              },
              {
                "fontSize": 16,
                "lineHeight": 24,
              },
              undefined,
              undefined,
              undefined,
              undefined,
              undefined,
            ]
          }
        />
      </Text>
    </View>
  </View>
</View>
`;
